### ISSUE TITLE
`gulp.src is not a function`

### ISSUE THREAD

**Issue by bendavis78** [stoeffel/gulp-param#17]
Getting this error when using gulp-param. For example:

```javascript
var gulp = require('gulp-param')(require('gulp'), process.argv);
var minifyCss = require('gulp-minify-css');

gulp.task('css', function() {
  gulp.src('./css/style.css')
  .pipe(minifyCss())
  .pipe(gulp.dest('css/style.min.css'));
});
```

```
[18:35:27] Starting 'css'...
[18:35:27] 'css' errored after 1.31 ms
[18:35:27] TypeError: gulp.src is not a function
    at Gulp.<anonymous> (/home/ben/Projects/allatx-api/gulpfile.js:5:8)
    at Gulp.wrappedTaskFunction (/home/ben/Projects/allatx-api/node_modules/gulp-param/index.js:30:29)
    at module.exports (/home/ben/Projects/allatx-api/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/home/ben/Projects/allatx-api/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/home/ben/Projects/allatx-api/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/home/ben/Projects/allatx-api/node_modules/orchestrator/index.js:134:8)
    at /home/ben/Projects/allatx-api/node_modules/gulp/bin/gulp.js:129:20
    at nextTickCallbackWith0Args (node.js:433:9)
    at process._tickCallback (node.js:362:13)
    at Function.Module.runMain (module.js:432:11)
```

**Comment by stoeffel** [comment]
arg. 1.0.1 is broken. The problem is how we wrap gulp. Will fix it today.
Thanks for reporting.

**Comment by stoeffel** [comment]
fixed and published with v1.0.3.
